frontend/src/components/LandingPage/TeamCard/TeamPage.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import TeamCard from './TeamCard';
+import { Container, Row, Col } from 'reactstrap';
+
+
+function TeamPage() {
+    return(
+  <>
+  <h1 className = "heading">Meet Our Team!</h1> <br />
+  <Container className = "container-width">
+  <Row> 
+  <Col lg = "4" md = "4" sm = "12" xs = "12">
```

Comment:
Part of coding practice is not putting space in element attributes.
`<Col md = "4" sm = "12">` should be `<Col md="4" sm="12">`.

Also because there are suppose to be 3 cards for each row on medium and large screens. The `lg` attribute is not necessary. 

Since each card are suppose to be on a separate line on small and extra small screens, including both `xs` and `sm` attributes is not necessary. Choose one or the other.